lib/src/signed_video_h26x_common.c

```diff
@@ -979,13 +979,6 @@ simply_hash(signed_video_t *self, const h26x_nalu_t *nalu, uint8_t *hash, size_t
     if (status == SV_OK) {
       // Finalize the ongoing hash of NALU parts.
       status = openssl_finalize_hash(self->crypto_handle, hash, false);
-      // For the first NALU in a GOP, the hash is used twice. Once for linking and once as reference
-      // for the future. Store the |nalu_hash| in |tmp_hash| to be copied for its second use, since
-      // it is not possible to recompute the hash from partial NALU data.
-      if (status == SV_OK && nalu->is_first_nalu_in_gop && !nalu->is_first_nalu_part) {
-        memcpy(self->gop_info->tmp_hash, hash, hash_size);
-        self->gop_info->tmp_hash_ptr = self->gop_info->tmp_hash;
```

Comment:
I can't see that `tmp_hash_ptr` nor `tmp_hash` is used anymore.  Remove them from `gop_info` struct.